falcon-tracer/falcon/core/events/handling/logger/event_logger.py

```diff
@@ -1,8 +1,18 @@
+import logging
 import ujson as json
 from event_parser import EventParser
 from falcon.core.events.handling.base_handler import BaseHandler
 
 class EventLogger(BaseHandler):
+    def __init__(self, console_output):
+        self.logger = logging.getLogger()
+        if console_output:
```

Comment:
With the new changes, we can get the `event_logger` here and let the configuration to `logging.ini`.

Therefore, the `console_output` parameter can be discarded since the same effect can be achieved by changing the `logging.ini`.